server/sonar-server/src/main/java/org/sonar/server/setting/ws/ValuesAction.java

```diff
@@ -154,8 +155,10 @@ private static ValuesRequest toWsRequest(Request request) {
       return Optional.empty();
     }
     ComponentDto component = componentFinder.getByKeyAndOptionalBranch(dbSession, componentKey, valuesRequest.getBranch());
-    if (!userSession.hasComponentPermission(USER, component) && !userSession.hasComponentPermission(SCAN_EXECUTION, component)) {
-      throw insufficientPrivilegesException();
+    if (!userSession.hasComponentPermission(USER, component) &&
+        !userSession.hasComponentPermission(SCAN_EXECUTION, component) &&
+        !userSession.hasPermission(OrganizationPermission.SCAN, component.getOrganizationUuid())) {
+        throw insufficientPrivilegesException();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make this line start at column 7. ([squid:IndentationCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Feh%2Ffix_scan_global_on_master&issues=AWG-Hi17JBf1BWcDXKL8&open=AWG-Hi17JBf1BWcDXKL8)

[ISSUE_KEY]: AWG-Hi17JBf1BWcDXKL8